Delivery-Service.API/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DeliveryService.Contracts;
+using DeliveryService.Database;
+using DeliveryService.Services;
+using DeliveryService.Models;
+
+namespace Delivery_Service.API.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
+    {
+        private static readonly string[] Summaries = new[]
+        {
+            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+        };
+
+        private readonly ILogger<WeatherForecastController> _logger;
+
+        public WeatherForecastController(ILogger<WeatherForecastController> logger)
+        {
+            _logger = logger;
+        }
+
+        // Task 1 ASP.Net
+
+        //  GET/menu
```

Comment:
for dish deleting it is Id need
so please add Id
do u lan to ad dish
if so - add endpoint
for other entitiesas well